OsmAnd-java/src/main/java/net/osmand/router/RoutingConfiguration.java

```diff
@@ -78,11 +78,36 @@
 		public final static String TAG = "osmand_dp";
 		public final static String DELETE_TYPE = "osmand_delete_point";
 		public final static String CREATE_TYPE = "osmand_add_point";
+		public final static double MAX_ANGLE_DIFF = 45;//in degrees
 
 		public DirectionPoint(Node n) {
 			super(n, n.getId());
 		}
-		
+
+		// get normalize angle in 0..360 degrees or Double.NaN if empty
+		public double getAngle() {
+			String angle = getTag("angle");
```

Comment:
make tag constant and use slightly another name "apply_direction_angle"

Reply:
ok